schedule.yaml

```diff
@@ -82,9 +82,77 @@ league:
     #   duration: -30   # seconds
 #
 knockout:
-  round_spacing: 300 # Delay between knockout rounds
-  final_delay: 180   # Extra delay before the finals
-  single_arena:
-    rounds: 0
-    arenas:
-    - main
+  static: true
+
+# Knockout: Heats, Quarters, Semis, Finals
+# Top 8 teams go straight through to quarters
+# Bottom 16 have to go through heats
+# Top 2 robots from each match go through
+# We design this for 24; in the 23 team case, there's a bye
+static_knockout:
+  teams_per_arena: 4
+
+  matches:
+    # Heats
+    0:
+      0:
+        arena: main
+        display_name: Heat 1
+        start_time: 2024-04-14 14:20:00+01:00
```

Comment:
The day schedule has these matches start at 14:15, is there a reason the first match is delayed from that?